### ISSUE TITLE
Berücksichtigung von `class:bicycle` und `traffic` (Case Siegfriedstraße)

### ISSUE THREAD

**Issue by tordans** [SupaplexOSM/OSM-Cycling-Quality-Index#6]
* https://www.osm-verkehrswende.org/cqi/map/?map=17.2%2F13.43528%2F52.46862
* https://www.openstreetmap.org/way/35551354
* Mapillary https://www.mapillary.com/app/?z=17.083427071675313&lat=52.4683014&lng=13.4343166&panos=true&dateFrom=2022-01-01&pKey=992667191896764&focus=photo&x=0.6952553836588279&y=0.5402476043552725&zoom=0

<img width="1058" alt="image" src="https://github.com/SupaplexOSM/OSM-Cycling-Quality-Index/assets/111561/7e1db991-8f06-46b3-b6bb-00b08c36bf9d">

Die Straße ist in Teilen sehr schlecht bewertet. Im Alltag ist sie aber ganz gut zu fahren.

Ich vermute, dass das nur über etwas wie https://wiki.openstreetmap.org/wiki/Key:traffic:hourly oder https://wiki.openstreetmap.org/wiki/Key:class:bicycle zu "korrigieren" / justieren ist, weil es von den baulichen Eigenschaften vermutlich gut abgebildet ist.

**Comment by SupaplexOSM** [comment]
Ja, das ist in der Tat mit "pure" OSM nicht abbildbar. Die nutzbare Breite beträgt hier etwa 4 Meter, was für eine mitgenutzte Auto-Fahrbahn mit Zweirichtungsverkehr verheerend ist. Das faktisch aber kaum Verkehr stattfindet, weiß der Algorithmus leider nicht. (Außerdem ist die `smoothness` nur als `intermediate` eingestuft, was ebenso zu Abwertung führt.)

Eine kleine Abhilfe könnte der subjektive Tag [`class:bicycle`](https://wiki.openstreetmap.org/wiki/Key:class:bicycle) schaffen, den ich plane als kleines Korrektiv mit zu implementieren. Würde man diesem Wegstück hier dann also `class:bicycle=1` oder `2` geben, würde er etwas ins Bessere rutschen.

Wir sollten aber auch nochmal darüber diskutieren, inwiefern wir das `traffic`-Tagging insbesondere zur Auswertung besonders verkehrsreicher oder verkehrsarmer Straßen nutzen können. Im Geodatenportal gibt es Verkehrsmengendaten aus 2019, aber leider nur für Hauptstraßen.
